ext/zend_test/test.c

```diff
@@ -435,6 +435,17 @@ static ZEND_FUNCTION(zend_test_is_string_marked_as_valid_utf8)
 	RETURN_BOOL(ZSTR_IS_VALID_UTF8(str));
 }
 
+static ZEND_FUNCTION(zend_get_zval_ptr)
+{
+	zval *v;
+
+	ZEND_PARSE_PARAMETERS_START(1, 1)
+			Z_PARAM_ZVAL(v)
+	ZEND_PARSE_PARAMETERS_END();
+
+	RETURN_LONG((zend_long) v);
```

Comment:
Testing the zval doesn't make sense. This will always be the same for CVs, even if the array wasn't reused. What would want is testing `Z_ARR_P`.

Reply:
thank you